src/components/navbar/index.js

```diff
@@ -2,42 +2,76 @@ import React from 'react';
 import styled from 'styled-components';
 
 const NavWrapper = styled.header`
-  background-color: lightpink;
-  padding: 10px;
-  color: black;
+  position: fixed;
+  background: #48C6DC;
+  padding: 1rem; 
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font: Roboto;
+  font-style: normal;
 
   nav {
     height: 100%;
   }
 `;
 
+const NavHeader = styled.a`
+  margin-left: 3vw;   
+  margin-top: 2vh;
```

Comment:
the vw unit makes this look a little odd on an ipad or a resized screen-- try using rem instead?